lib/load-adapter-from-app-dependencies.js

```diff
@@ -39,18 +39,12 @@ module.exports = function loadAdapterFromAppDependencies(adapterPackageName, dat
   //  └─  └  ┴└─└─┘┴ ┴  ┴ ┴┴  ┴    ┘└┘└─┘─┴┘└─┘────┴ ┴└─┘─┴┘└─┘┴─┘└─┘└─┘  └  └─┘┴─┘─┴┘└─┘┴└─  ─┘
 
   // Since it is unknown so far, try and load the adapter from `node_modules`
-  sails.log.verbose('Loading adapter (`%s`) from this app\'s `node_modules/` directory...', adapterPackageName);
-
-  // Before trying to actually require the adapter, determine the path to the module
-  // relative to the app we're loading:
-  var userlandDependenciesPath = Path.resolve(sails.config.appPath, 'node_modules');
-  var adapterPackagePath = Path.join(userlandDependenciesPath, adapterPackageName);
 
 
   // Now try to require the adapter from userland dependencies (node_modules of the sails app).
   var adapter;
   try {
-    adapter = require(adapterPackagePath);
+    adapter = require(adapterPackageName);
```

Comment:
No, you can install whatever you wish and it will load just fine. What's the difference between: `require('package')` and `require('./some/folder/package')`.

If `package` is installed on project root, node will go up and will find the `package` so we're good.